.github/workflows/auto-translate-docs-reusable.yml

```diff
@@ -0,0 +1,122 @@
+            2. Get the original PR information using `gh pr list --state merged --limit 5` or `gh pr view <PR_NUMBER>` if PR number is provided.
+            3. Identify files that were changed in the merged PR in the "docs/en-us/" directory.
+            4. For each changed file, create or update the corresponding Japanese translation in "docs/ja-jp/".
+            5. Maintain the same directory structure and filename in the Japanese docs.
+            6. Create a new branch and pull request with the translations by using the PR template.
+
+            ## Requirements
+            - Translate all English text to natural, professional Japanese.
+            - Preserve all MDX/Markdown formatting exactly (headers, lists, code blocks, links).
+            - Keep code examples, URLs, and technical terms in English.
+            - Ensure proper UTF-8 encoding for Japanese characters.
+            - Create "docs/ja-jp/" directory structure if it doesn't exist.
+            - For existing translations: Preserve unchanged content and only update sections that correspond to changes in the English version.
+            - For new translations: Create complete Japanese translations of the entire document.
+            - Add the following import and component AFTER the main title (# heading) and BEFORE any other content of each translated document:
+              ```
+              import TranslationBanner from '/src/components/_translation-ja-jp.mdx';
+
+              <TranslationBanner />
+              ```
+              IMPORTANT: Place this immediately after the main title line, not before it.
+            - In the front matter properties, change `displayed_sidebar: docsEnglish` to `displayed_sidebar: docsJapanese`.
+
+            ## Process
+            1. Use git to identify which files were modified in the merged PR and get the PR details.
+            2. For each modified English documentation file:
+               a. Check if a Japanese translation already exists in "docs/ja-jp/".
+               b. If a translation exists: Use `git diff` to identify only the changed sections in the English file.
+               c. If a translation exists: Update only the changed sections in the Japanese file, preserving unchanged content.
+               d. If no translation exists: Create a completely new Japanese translation.
+            3. Create a new branch with the format "ja-jp/<ORIGINAL_PR_BRANCH_NAME_FOR_ENGLISH_PR>", using chain case (-) to separate words.
+            4. Create high-quality Japanese translations (full or partial based on changes).
+            5. Save translations in docs/ja-jp/ with identical file structure.
+            6. Commit the translations with a descriptive message indicating whether it's a new translation or an update.
+            7. Push the branch by using: `git push origin <branch-name>`.
+            8. Create a pull request by following the PR template instructions section below.
```

Comment:
Some Scalar projects use GitHub projects to specify target versions for a PR. Don't we need to consider it when creating a PR in this workflow?

Reply:
Yes! Manual select might be annoying. I think it would be helpful if it's automated. But, it's just an idea and I don't insist on it at all.